.circleci/config.yml

```diff
@@ -629,6 +620,28 @@ commands:
           command: echo "machine api.mapbox.com login mapbox password $SDK_REGISTRY_TOKEN" >> ~/.netrc
       - inject-mapbox-public-token
 
+  login-google-cloud-platform:
+    steps:
+      - run:
+          name: Log in to Google Cloud Platform
+          command: |
+            echo "${GOOGLE_CLOUD_KEY_FILE}" > secret.json
+            gcloud auth activate-service-account --key-file secret.json --project mapbox-maps-ios-dc24c
+            rm secret.json
+
+  get-test-results-from-firebase:
+    parameters:
+      scheme:
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: Retrieve TestResult from gcloud for << parameters.scheme >>
+          command: |
+            testResult=$(gsutil ls gs://test-lab-s350khs08d6wq-h1zw34xwm6w78/<< parameters.scheme >>-${CIRCLE_BUILD_NUM}/iphone11-13.6-en-portrait)
```

Comment:
Yes,  that's the project in firebase.  Do you think it's unsafe if we use it directly?

Reply:
I think it is just more configurable. We would be able to change the firebase project without any changes in the source code. 
My general motivation is that it is okay to hardcode Apple related information (including team-wide like applemachine account name or team identifier) but I would like to avoid any unusual constants like project identifier.

Reply:
Will extract it to environment variable.